### PR TITLE
[1LP][RFR] Use the externalized MIQ REST API Client

### PR DESCRIPTION
**Pull request by mkoura**
Updated #3293 from @mfalesni 

**PRT errors**:
`TimedOutError`, `NoSuchElementException`, `No such 'providers' matching query`, `NotFound: The resource could not be found. (HTTP 404)`
Nothing related to REST API client replacement, can be merged :-)

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/cloud/instance/__init__.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 from navmazing import NavigateToSibling, NavigateToAttribute
 from selenium.common.exceptions import NoSuchElementException
+from fixtures.pytest_store import store
```
Comment (mshriver) [inline cfme/cloud/instance/__init__.py]: See utils.appliance.get_or_create_current_appliance(), I think that may be more appropriate to use here.

File: cfme/tests/containers/test_static_custom_attributes.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import pytest
+from manageiq_client.api import APIException
```
Comment (mshriver) [inline cfme/tests/containers/test_static_custom_attributes.py]: space these out for formatting
```
import pytest

from manageiq_client.api import APIException

from utils.(...)
```

File: utils/appliance/__init__.py [hunk 3]
```diff
@@ -5,6 +5,7 @@
 import re
 import socket
 import yaml
+from manageiq_client.api import ManageIQClient as MiqApi  # noqa
```
Comment (mshriver) [inline utils/appliance/__init__.py]: Why noqa here?

**Comment by mshriver** [comment]
I agree with your assessment of PRT failures from run 9537, they're unrelated to your API client changes.

**Pull request merged**